### ISSUE TITLE
Content empty for "Scaling Filebench workloads"

### ISSUE THREAD

**Issue by mengwanguc** [filebench/filebench#135]
The content of "Scaling Filebench workloads" page is empty.....
Is it indended, or did I miss something?

The link: https://github.com/filebench/filebench/wiki/Scaling-Filebench-workloads

I thought It's supposed to be a detailed discussion on how to scale the Filebench workload......

**Comment by sectorsize512** [comment]
Yeah, the docs were never finished :( Maybe will have some time to work on it this year.
